pandas/_libs/groupby_helper.pxi.in

```diff
@@ -428,14 +428,20 @@ def group_rank_{{name}}(ndarray[float64_t, ndim=2] out,
     is_datetimelike : bool
         unused in this method but provided for call compatibility with other
         Cython transformations
-    ties_method :  {'keep', 'top', 'bottom'}
-        * keep: leave NA values where they are
-        * top: smallest rank if ascending
-        * bottom: smallest rank if descending
+    ties_method : {'average', 'min', 'max', 'first', 'dense'}
+        * average: average rank of group
```

Comment:
Perhaps clearer to say `average rank of tied values` - saying "group" can be confused with the larger Group object. Similar change needed for below points